_usage/command-line.md

```diff
@@ -1,212 +1,121 @@
-	Delimiters can be multi-char strings appropriately escaped for your shell.
-	Default column delimitor is <tab>; default row delimiter is <newline>
-```
-
-As you can see, the syntax is relatively easy and straightforward.
+As previously stated, currently you can not connect using the SSMS Object Explorer.
 
-If you want to see which compile time settings are available in your client
-interface, use the -C flag as shown in the next listing:
 
-```bash
-[user@host]$ tsql -C
-Compile-time settings (established with the "configure" script)
-                            Version: freetds v1.1.20
-             freetds.conf directory: /etc
-     MS db-lib source compatibility: yes
-        Sybase binary compatibility: yes
-                      Thread safety: yes
-                      iconv library: yes
-                        TDS version: auto
-                              iODBC: no
-                           unixodbc: yes
-              SSPI "trusted" logins: no
-                           Kerberos: yes
-                            OpenSSL: no
-                             GnuTLS: yes
-                               MARS: yes
+### Using `tsql` to connect
 
-```
+`tsql` is a command line tool that is shipped as part of FreeTDS. It allows you
+to connect and to interact with Babelfish (or any other TDS enabled data source)
+from your Linux terminal.
 
-Now that the syntax and the client settings are clear, we can easily connect to the server:
+Here is an example how to connect to Babelfish with `tsql`:
 
 ```bash
 tsql -S database.example.com -p 1433 -U postgres -P secretpassword
 ```
 
-A word of caution: The tsql binary allows you to send
-the password as part of the command. This normally results in a security
-problem. Fortunately, tsql replaces the command name so that the password is not
-visible in the process table, as shown in the next listing:
-
-```bash
- 425705 pts/5    Ss     0:00  |   \_ bash
- 426036 pts/5    S+     0:00  |   |   \_ tsql -H database.example..com -p 1433 -U postgres -P ************* -D postgres
-```
-
-If your connection has been established successfully, you should find yourself in
-a command line:
-
-```bash
-1> SELECT 1+1
-2> GO
-
-2
-(1 row affected)
-
-1> SELECT 1+1 AS x UNION ALL SELECT 2+2
-2> GO
-x
-2
-4
-(2 rows affected)
-```
-
-There are a couple of things worth mentioning here: First of all, we have to use
-<code>GO</code> to make Babelfish execute the command. This is just like in standard MS SQL,
-as well as in Sybase. But there is more, which might be alien to users
-experienced with PostgreSQL: <code>SELECT 1+1</code> does not yield a column title, so to
-ensure that a title is provided, we need to alias the column. You need to be aware
-of such small details in order to ensure that the code written stays portable, and still
-works with Microsoft SQL Server as well as PostgreSQL.
-
+Like in `sqlcmd`, the command `USE` selects a database, and `GO` is used to send
+a batch of SQL statements to the server.
```

Comment:
Instead of sharing a snippet of syntax, it might be more helpful to send the user to the official project docs:

For information about using `tsql`, visit the project documentation at: [https://www.freetds.org/userguide/]

Reply:
Also agree with that, done.